x-pack/plugins/beats_management/public/router.tsx

```diff
@@ -45,8 +45,12 @@ export class AppRouter extends Component<RouterProps, RouterState> {
 
   public async componentWillMount() {
     if (this.state.loadingStatus === 'loading') {
-      await this.props.beatsContainer.reload();
-      await this.props.tagsContainer.reload();
+      try {
+        await this.props.beatsContainer.reload();
+        await this.props.tagsContainer.reload();
+      } catch (e) {
+        // TODO in a furture version we will better manage this "error" in a returned arg
```

Comment:
Based on the description we expect a 404 here sometimes? Can we detect and ignore that specific error and rethrow others please?

Reply:
In a future release I’ll be refining how we handle errors throughout BeatsCM before GA, so refining now would be a wasted effort however non-ideal it might be